### PR TITLE
community/quassel: improve

### PR DESCRIPTION
**Pull request by tcely**
- fix license
- add icu-dev
- use CXXFLAGS

### CODE REVIEW

File: community/quassel/APKBUILD [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 pkgname=quassel
 pkgver=0.13.1
-pkgrel=2
+pkgrel=3
 url="https://www.quassel-irc.org/"
 pkgdesc="Distributed IRC client - metapackage"
 arch="all"
```
Comment (maxice8) [inline community/quassel/APKBUILD]: license is not SPDX and is ambiguous

File: community/quassel/APKBUILD [hunk 2]
```diff
@@ -17,7 +17,7 @@ depends_client="$pkgname-common"
 depends_mono="$pkgname-common"
 depends_server="$pkgname-common qt5-qtbase-postgresql qt5-qtbase-sqlite"
 depends_server_openrc=""
```
Comment (maxice8) [inline community/quassel/APKBUILD]: empty varialbes can be removed for brevity

Comment (tcely) [reply community/quassel/APKBUILD]: brevity is not a ranking concern

**Comment by Ikke** [comment]
@starwarsfan Can you review this? Thanks

**Comment by starwarsfan** [comment]
Sry for the delay. Ok for me, thx for the patch!

**Pull request closed without merging**